nano/rpc/rpc_handler.cpp

```diff
@@ -3970,6 +3974,7 @@ void nano::rpc_handler::wallet_pending ()
 	const bool source = request.get<bool> ("source", false);
 	const bool min_version = request.get<bool> ("min_version", false);
 	const bool include_active = request.get<bool> ("include_active", false);
+	const bool include_only_confirmed = request.get<bool> ("include_only_confirmed", false);
```

Comment:
We'll need an RPC test for this flag.

Reply:
Added a test for it, currently there isn't one for the existing `include_active` though.